src/Tribe/RSVP.php

```diff
@@ -408,14 +408,15 @@ public function generate_tickets( ) {
 		}
 
 		$has_tickets = $event_id = false;
+
 		/**
 		* RSVP specific action fired just before a RSVP-driven attendee ticket for an event is generated
-		* 
+		*
 		* @param $event_id ID of event
 		* @param $_POST post paremeters comes from RSVP Form
 		*/
-		do_action( 'tribe_tickets_rsvp_before_tickets_generated', $event_id, $_POST);
-		
+		do_action( 'tribe_tickets_rsvp_before_tickets_generated', $event_id, $_POST );
```

Comment:
Seems a little odd to pass a superglobal that the callback can access anyway (though I guess we might switch it up in future and pass a modified/untainted array). More importantly though, isn't `$event_id` always going to be false at this point?